src/hooks/useActiveUser.tsx

```diff
@@ -0,0 +1,11 @@
+import { useSelector, useStore } from "react-redux";
+import { getActiveUserTag, getWSControllerURL } from "app/selectors";
+
+export default function useActiveUser() {
```

Comment:
Nice.......but ;)

This will return the active user on the root controller for the application, not the controller that the model is on. While these CAN be the same they don't have to be, like when you register an additional controller. So I'm not sure that this is going to be the best approach for this problem as you need to query the active user for every row in the model list.

Reply:
Created separate issue here: https://github.com/canonical-web-and-design/jaas-dashboard/issues/1066